### PR TITLE
Start implementing time toolbar window

### PR DESCRIPTION
**Pull request by marijnvdwerf**

### CODE REVIEW

File: src/openloco/windows/TimePanel.cpp [hunk 1]
```diff
@@ -0,0 +1,451 @@
...
+                sub_439AB6(window);
+                break;
+        }
+    }
+
+    static void sub_43A67F(ui::window* self, widget_index widgetIndex)
+    {
+        auto skin = objectmgr::get<interface_skin_object>();
+
+        if (isNetworked())
```
Comment (AaronVanGeffen) [inline src/openloco/windows/TimePanel.cpp]:
```suggestion
        if (!isNetworked())
```

File: src/openloco/windows/TimePanel.cpp [hunk 2]
```diff
@@ -0,0 +1,451 @@
...
+        {
+            // TODO: remove image id addition
+            dropdown::add(0, string_ids::str_1287, { (uint32_t)skin->img + interface_skin::image_ids::toolbar_menu_map_north + gCurrentRotation, string_ids::menu_map });
+            dropdown::set_highlighted_item(0);
+        }
+        else
+        {
+            // TODO: remove image id addition
+            dropdown::add(0, string_ids::str_1287, { (uint32_t)skin->img + interface_skin::image_ids::phone, string_ids::str_1716 });
```
Comment (AaronVanGeffen) [inline src/openloco/windows/TimePanel.cpp]:
```suggestion
            dropdown::add(0, string_ids::str_1287, { (uint32_t)skin->img + interface_skin::image_ids::phone, string_ids::str_1716 });
             dropdown::add(1, string_ids::str_1287, { (uint32_t)skin->img + interface_skin::image_ids::toolbar_menu_map_north + gCurrentRotation, string_ids::menu_map });
```

File: src/openloco/objects/competitor_object.h [hunk 3]
```diff
@@ -0,0 +1,23 @@
+#pragma once
+
+#include "../localisation/stringmgr.h"
+
+namespace openloco
+{
+#pragma pack(push, 1)
+    struct competitor_object
+    {
+        string_id var_00;        // 0x00
```
Comment (duncanspumpkin) [inline src/openloco/objects/competitor_object.h]: Var_00 is always the name of the object

Comment (duncanspumpkin) [reply src/openloco/objects/competitor_object.h]: Have you checked this against locotools for any assistance on naming vars

Comment (duncanspumpkin) [reply src/openloco/objects/competitor_object.h]: Yeah sorry I wrote another message saying all the information that could be gleamed from LocoTools has been and to ignore my comment but it ended up in a pending review by accident.

**Comment by marijnvdwerf** [comment]
Should be everything implemented

**Pull request merged**